pkgs/applications/audio/musly/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, cmake, eigen, libav_all }:
+stdenv.mkDerivation rec {
+  name = "musly-${version}";
+  version = "0.2";
```

Comment:
There's no tag named 0.2, since you pick a random git revision you should put the commit-date here instead, the form `unstable-YYYY-MM-DD`  is usually quite accepted.